src/scrape_up/MediEncyclopedia/mediencyclopedia.py

```diff
@@ -0,0 +1,72 @@
+from html.parser import HTMLParser
+import bs4
+from bs4 import BeautifulSoup
+import requests
+
+"""
+        Create an instance of `MediEncyclopedia` class.
+
+    ```python
+    ency = MediEncyclopedia()
+    ```
+
+    | Methods               | Details                                                             |
+    | --------------------- | ------------------------------------------------------------------- |
+    | `.scrapebyurl()`      | Returns the medical dictation of associated topic url               |
+    | `.byletter()`         | Returns the list of medical relics starting with a particular letter|
+    
+
+    """
+
+
+class MediEncyclopedia:
+
+  def __init__(self):
+    print(self)
+
+  def scrapebyurl(self,url):
+    """Returns the medical data including references, review dates, content, topic-head and so on"""
+
+    try:
+      content = requests.get(url)
+      soup = BeautifulSoup(content.content, 'html.parser')
+      headline = soup.find('h1',attrs={'class':'with-also'}).text
+      article = soup.find('article')
+      for script in article(["script", "style"]):
+        script.extract()
+      text = article.get_text().replace('Browse the Encyclopedia','').replace('Clinical Trials','')
+      text = text.replace('To use the sharing features on this page, please enable JavaScript.','')
+
+      return [headline,text]
+    except:
+      return None  
+
+
+
+
+  def byletter(self, character):
+    """Returns the list of medical encyclopedia terms starting with a particular english character"""
+    try:
+      character = character.upper()
+      chk = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+      if character not in chk:
+        return None
+      url = 'https://medlineplus.gov/ency/encyclopedia_{}.htm'.format(character)
```

Comment:
I think the `URL` defined here is incorrect. At the end of the URL, it must be `.html` right?

Check once, and refactor your PR.

Reply:
Hi, the url here is correct, the page loads in htm only
<img width="1470" alt="Screenshot 2023-07-15 at 7 40 09 PM" src="https://github.com/Clueless-Community/scrape-up/assets/93859359/a392fd7d-d523-4491-85ba-621bb8b3e319">